weechat-android/build.gradle

```diff
@@ -1,9 +1,9 @@
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
The build tools plugin seems like it's only available from jcenter. Here `jcenter` is a superset of the packages available from `mavenCentral`